jupyter_telemetry/eventlog.py

```diff
@@ -44,15 +30,12 @@ class EventLog(Configurable):
     """
     Send structured events to a logging sink
     """
-    # Traitlets can't take non-pickleable objects, so can't pass in
-    # logging handlers directly. This is a temporary hack we should fix?
-    handlers_maker = Callable(
+    handlers_list = HandlersList(
```

Comment:
naming nitpick: Unless there's a practice to name traits this way, Is just `Handlers()` not sufficient?

Reply:
This is changed in #17